Android/RenderScriptV8/samples/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="BuildOutput" value="../output" />
```

Comment:
This will install the test package into the NuGet cache. You might want to set these properties as well: https://github.com/xamarin/XamarinComponents/blob/master/Android/Kotlin/samples/Directory.Build.props#L4